install_files/ansible-base/roles/restrict-direct-access/templates/rules_v4

```diff
@@ -5,36 +5,36 @@
 :LOGNDROP - [0:0]
```

Comment:
As this file is updated only when the install playbook is run, its changes will not be reflected on many production systems. This means that there will be configuration drift (albeit comments-only) between existing and new instances that may complicate support and automated tests. IMO it would be worth deferring these changes until the next mandatory install playbook run (currently the trusty->focal migration, but could happen before then).

Reply:
This makes a lot of sense to me, I suspected the iptables rules had potential to be a special case _somehow_.

Reply:
I removed all changes to the `install_files/` directory. https://github.com/freedomofpress/securedrop/pull/5448#pullrequestreview-467929827